src/sites/twitch-twilight/modules/chat/index.js

```diff
@@ -425,6 +431,59 @@ export default class ChatHook extends Module {
 			for(const inst of instances)
 				this.fixPinnedCheer(inst);
 		});
+
+		this.TabComplete.ready((cls, instances) => {
+			for(const inst of instances)
+				this.setupTabCompletion(inst);
+		});
+		this.TabComplete.on('mount', this.setupTabCompletion, this);
+	}
+
+	getResultsForSets(input, userID, channelID, renderEmoteSuggestion) { // eslint-disable-line
+		const search = input.substring(1),
+			results = [],
+			emoteSets = this.chat.emotes.getSets(userID, undefined, channelID);
```

Comment:
You probably want to be using `emotes.getEmotes(user_id, user_login, room_id, room_login)` here. It returns an object. Keys are the emote names. Values are the emote data.

This way, it'll only show the first emote and the one that will be used if there's a name collision. It also simplifies your code here as you don't have to iterate emoteSets too.